content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Agent Transport for logs
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
+
+Since Agent v6.19+/v7.19+, the default transport is HTTPS with compression instead of TCP.
+At agent start, if log collection is enabled, the agent runs a HTTPS connectivity test:
+
+* if successful, then the agent uses compressed HTTPS transport
+* otherwise the agent fallbacks to TCP transport
+
+This connectivity test mechanism is only running at Agent startup. If the connectivity is failing for both HTTPS and TCP at Agent startup time, then the Agent uses TCP transport until its next restart (and then rerun a new connectivity test)
```

Comment:
So if we fail TCP test we use TCP test ? 🤔 
```suggestion
This connectivity test mechanism is only performed when the Agent starts. If it is fails for both HTTPS and TCP, then the Agent uses TCP transport until its next restart (and then rerun a new connectivity test).
```

Reply:
yes... this wording might be a bit confusion... the "connectivity test" only test HTTPS. which means that if there is no connectivity at all (both TCP and HTTP) the agent will use TCP once the connectivity will be back.
I'll rephrase that.

Reply:
updated, tell me what you think!